scheduler/core/eventsqueue/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .queue import EventQueue
+from .events import (Event, Interruption, Blockage,
+                     WeatherChange, ResumeNight, Fault,
+                     Rto0s)
```

Comment:
Why the `Rto0s`, out of curiosity, instead of `Rtoos` or something?

Let's not worry about rapid ToOs just yet, and focus on one event... I think RToOs should come in as events that are either new programs or changes to programs, but we should discuss this and just get the event model working first.

Reply:
Agree!